### ISSUE TITLE
I wonder if the softmax layer is added to this model

### ISSUE THREAD

**Issue by Jellymancbd** [dragen1860/GCN-PyTorch#6]
the function masked_loss() may have not softmax.
